python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -358,12 +358,13 @@ def _report_current_usage(self) -> None:
         pending_usage = self._resource_manager.get_global_pending_usage()
         limits = self._resource_manager.get_global_limits()
         resources_status = (
-            "Running. Resources: "
+            # TODO(scottjlee): Add dataset name/ID to progress bar output.
+            "Running Dataset. Resource usage: "
             f"{running_usage.cpu:.4g}/{limits.cpu:.4g} CPU, "
             f"{running_usage.gpu:.4g}/{limits.gpu:.4g} GPU, "
             f"{running_usage.object_store_memory_str()}/"
-            f"{limits.object_store_memory_str()} object_store_memory "
-            "(pending: "
+            f"{limits.object_store_memory_str()} object store "
+            "(⏳: "
```

Comment:
can we hide the pending section if all are 0? (forgot to mention in the previous PR)